### PR TITLE
Expanded section on system property changes with advice on how to use those properties safely

### PR DESCRIPTION
**Pull request by mvenzke-axway**
https://deploy-preview-1675--axway-open-docs.netlify.app/docs/apim_reference/system_props/

This info existed before in a KB - https://support.axway.com/en/articles/article-details/id/167287/do/search

However, it is a common source of upgrade problems for customers to have made changes to the system jvm.xml file and then just lose them during upgrade so we should provide some more advice on this page.

### CODE REVIEW

**Comment by mvenzke-axway** [comment]
This info existed before in a KB - https://support.axway.com/en/articles/article-details/id/167287/do/search

However, it is a common source of upgrade problems for customers to have made changes to the system jvm.xml file and then just lose them during upgrade so we should provide some more advice on this page.

**Comment by andreamussap** [comment]
@maherb4 @dshielaxway Please review/ acknowledge the change from technical perspective, so I can perform the editorial review. Thanks.

File: content/en/docs/apim_reference/system_props.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ Traditionally, the JVM.xml file has allowed customers to tweak the behavior of t
...
+For settings that are intended to apply to a single API Gateway instance, the settings should be placed in the extension jvm.xml file at `/apigateway/groups/[group X]/[instance Y]/conf/jvm.xml`, while settings that need to apply to all API Gateway instances on the local machine should be placed in the extension jvm.xml file at `/apigateway/conf/jvm.xml`.  By default, those files do not exist, so they need to be created the first time.  Both of the extension jvm.xml files require that all settings be wrapped in a single `<ConfigurationFragment>` tag, for example:
+
+```
+<ConfigurationFragment>
+<!-- Your notes, e.g. a link to this page. -->
+<VMArg name="-Dsome.property=some.value"/>
+</ConfigurationFragment>
+```
+
+It is a best practice to leave yourself XML comments reminding you both what the setting does and why you want that change in behavior.
```
Comment (dshielaxway) [inline content/en/docs/apim_reference/system_props.md:25]: It is best practice to add XML comments as a reminder for what the setting does, and why the change in behaviour is required.

File: content/en/docs/apim_reference/system_props.md [hunk 2]
```diff
@@ -11,6 +11,19 @@ Traditionally, the JVM.xml file has allowed customers to tweak the behavior of t
 * All subsequent tables contain newly added properties introduced in that specific version, or service pack, and the assumption is that properties listed in previous tables are inherited.
 * Note that because of fix ports, some properties might be duplicated among the three versions: 7.5.3, 7.6.2, and 7.7.
 
+No changes should ever be made to the system jvm.xml file located at `/apigateway/system/conf/jvm.xml` because this file is replaced during both updates and upgrades, so any changes made there will be lost.  The system jvm.xml file defines include points that allow you to separate custom configuration from the system jvm.xml file in a way that will not get lost when the system jvm.xml file is replaced.
```
Comment (maherb4) [inline content/en/docs/apim_reference/system_props.md:14]: Maybe this should be a warning message or a note to make it stand out.

It could be re-worded to say, customers should never edit the system jvm.xml.....

**Review by maherb4 — APPROVED** [review]
Overall a helpful addition

File: content/en/docs/apim_reference/system_props.md (hunk 2 above)
Comment (andreamussap) [reply content/en/docs/apim_reference/system_props.md:14]: Leave it with me. I'll see how it looks when I update the page more. Thanks

**Review by andreamussap — APPROVED** [review]
